src/main/java/com/recurly/v3/QueryParams.java

```diff
@@ -1,13 +1,7 @@
 package com.recurly.v3;
 
-import com.google.gson.annotations.SerializedName;
```

Comment:
This is a generated file, I need to add the preamble to the top. I'll go ahead and remove these from the generator as well. Thanks

Reply:
Oops, yeah missed that.

Reply:
I'll remove from this commit